src/options/mkoptions.py

```diff
@@ -116,10 +116,18 @@
 TPL_PUSHBACK_PREEMPT = 'extender->pushBackPreemption({});'
 
 TPL_HOLDER_MACRO_ATTR = "  {type} {name};\\\n"
-TPL_HOLDER_MACRO_ATTR += "  bool {name}__setByUser__ = false;"
+TPL_HOLDER_MACRO_ATTR += "  bool {name}__setByUser = false;"
 
 TPL_HOLDER_MACRO_ATTR_DEF = "  {type} {name} = {default};\\\n"
-TPL_HOLDER_MACRO_ATTR_DEF += "  bool {name}__setByUser__ = false;"
+TPL_HOLDER_MACRO_ATTR_DEF += "  bool {name}__setByUser = false;"
+
+TPL_DECL_SET_DEFAULT = 'void default_{name}(Options& opts, {type} value);'
```

Comment:
Would `setDefault_` ... be a better name here?

Reply:
using `setDefault{OptionName}` now